services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession
+from django.db.models import QuerySet
+import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        return queryset.filter(show_time__date=session_date)
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> object:
+    return MovieSession.objects.get(id=movie_session_id)

```

Comment:
if object doesn't exist u will get an internal server error, if u don't handle it in the middleware u should use
get_object_or_404() func instead